packages/langium/src/lsp/completion/completion-provider.ts

```diff
@@ -239,6 +242,9 @@ export class DefaultCompletionProvider implements CompletionProvider {
     }
 
     protected completionForKeyword(keyword: ast.Keyword, context: AstNode | undefined, acceptor: CompletionAcceptor): MaybePromise<void> {
+        if (keyword.value.match(/^[\W]/) && keyword.value.length === 1 && !this.languageMetadata.showNonAlphabeticKeywords) {
```

Comment:
The name `showNonAlphabeticKeywords` makes it a bit hart to think about it:
1. `show` -> ok
2. `non-alphabetic-keywords`
   1. `keyword` makes sense
   2. `alphabethic`, mhhh ok like ABC
   3. `non` = negating the term before
3. now try to read `!non-alphabetic-keywords` , what is it?

Proposal: `showOnlyAlphabeticKeywords`... ok, when NOTTING it is a bit the same. ^^*... so even I am not sure about this solution...

Reply:
That would remove the need for negation in the if statement